### PR TITLE
Clean up testing code of kubectl cmd section

### PR DESCRIPTION
**Pull request by weikhor**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Clean up testing code of kubectl cmd section
#### Which issue(s) this PR fixes:

None
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go [hunk 1]
```diff
@@ -443,10 +439,10 @@ func TestAnnotateErrors(t *testing.T) {
 				return
 			}
 			if bufOut.Len() > 0 {
-				t.Errorf("buffer should be empty: %s", string(bufOut.Bytes()))
+				t.Errorf("buffer should be empty: %s", bufOut.String())
 			}
 			if bufErr.Len() > 0 {
-				t.Errorf("buffer should be empty: %s", string(bufErr.Bytes()))
+				t.Errorf("buffer should be empty: %s", bufOut.String())
```
Comment (brianpursley) [inline staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go:445]: This should be `bufErr` not `bufOut`

**Review by brianpursley — CHANGES_REQUESTED** [review]
Hi @weikhor thanks for the PR.

I left you one comment.

Also, can you squash your commits to a single commit?  
https://www.kubernetes.dev/docs/guide/github-workflow/#squash-commits

**Comment by weikhor** [comment]
/remove-area cloudprovider code-generation conformance dependency e2e-test-framework ipvs kubeadm kubectl kubelet

**Comment by weikhor** [comment]
/remove-area area/provider/gcp area/release-eng area/stable-metrics

**Comment by weikhor** [comment]
/remove-sig api-machinery  apps architecture auth cloud-provider  cluster-lifecycle network node release scalability scheduling security  storage windows

**Comment by KnVerey** [comment]
The change looks good to me, but I see that the commit is marked as co-authored by @sss-ng who has not participated on this PR so far. Can you confirm that that is correct?

**Comment by sss-ng** [comment]
@KnVerey that is correct. I have no idea how my username is listed there. Any ideas?

**Comment by KnVerey** [comment]
Agreed, it probably happened during the rebase. @weikhor can you please re-do the commit to remove the incorrect co-author? While you're at it, a more specific commit message (closer to your PR description) would be a nice bonus. Thanks!

**Comment by weikhor** [comment]
> Agreed, it probably happened during the rebase. @weikhor can you please re-do the commit to remove the incorrect co-author? While you're at it, a more specific commit message (closer to your PR description) would be a nice bonus. Thanks!

Have updated changes accordingly, Thank for review.

**Pull request merged**